.golangci.yml

```diff
@@ -228,3 +228,10 @@ linters-settings:
     min-complexity: 154
   nakedret:
     max-func-lines: 15
+
+issues:
+  exclude-rules:
+    - path: test/nri/nri_test.go
+      linters:
+        - dupl
+      text: "lines are duplicate of"
```

Comment:
Perhaps you can work around these (by moving some code into common functions or methods) instead of disabling it.

Reply:
Those were 'false' positives, due to structural similarity of some of the test cases. But I'll take another look to see if I can extract some of the common bits and get rid of the duplication error, by somehow 'hyperparametrizing' the offending tests and folding them into one test function.

Reply:
Okay, it's not going to be a beauty-contest winner, but I managed to fold both offending pairs into a parametrized single function and get rid of both false positive duplicate errors and the need for duplicate error suppression.